modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js

```diff
@@ -81,7 +81,11 @@ const useFormSubmit = ({apiRef, containerRef}) => {
 							}
 
 							if (submittable) {
-								Liferay.Util.submitForm(event.target);
+								const form = event.target;
+
+								if (form.nodeName === 'FORM') {
```

Comment:
How come we are adding these extra checks? It seems like the Util version simply just calls `.submit()` without checking anything. I think we can do the same

Reply:
Previously when I just had `event.target.submit()` those regressions were failing a couple weeks ago. What happens when `event.target` isn't a `form` element? I think that AUI handles it under the hood whereas JS doesn't.

Reply:
Hmm, but then the question is why would the old implementation work? The old implementation with `Util.submitForm` was directly calling `.submit()` without any checks.

```
submitForm(form) {
	form.submit();
},
```


If this code works, then we shouldn't need the custom checks, unless something else is going on